docs/technology/glossary.md

```diff
@@ -23,17 +21,16 @@ validating information.
 
 **Stake**: locking funds in a staking pool in order to secure the network and receive rewards.
 
-**Delegate**: locking the funds in the “legacy” delegation contract in order to secure the network
-and receive rewards.
+**Delegate**: contribute to the network security by delegating 1 EGLD or more towards a staking provider that operates 
```

Comment:
the idea behing this was the "Delegate"-Tab in the wallet. There are 3 tabs there: stake (staking providers), delegate (legacy) and validate (run nodes).

-> so maybe change to: delegating **10** EGLD or more towards **the Elrond Community Delegation contract** community